functional/vfio/run.sh

```diff
@@ -94,32 +98,63 @@ Usage: $0 [-h] [options]
     Description:
         This script runs a kata container and passthrough a vfio device
     Options:
-        -h,         Help
-        -s <value>, Set sandbox_cgroup_only in the configuration file
+        -h,          Help
+        -m <string>, Specify kata-runtime machine type for qemu hypervisor
+        -p <string>, Specify kata-runtime hypervisor
+        -s <value>,  Set sandbox_cgroup_only in the configuration file
 EOF
 }
 
 setup_configuration_file() {
+	clh_config_file="configuration-clh.toml"
```

Comment:
Nit: these variables could all be `local`.

Reply:
@jodh-intel, applied these changes